balancer/grpclb/grpclb_remote_balancer.go

```diff
@@ -53,7 +54,7 @@ func (lb *lbBalancer) processServerList(l *lbpb.ServerList) {
 	lb.serverListReceived = true
 
 	// If the new server list == old server list, do nothing.
-	if reflect.DeepEqual(lb.fullServerList, l.Servers) {
+	if cmp.Equal(lb.fullServerList, l.Servers, cmp.Comparer(proto.Equal)) {
```

Comment:
How is this different from `if proto.Equal(lb.fullServerList, l.Servers) {`?

Reply:
Hmm, good question

Reply:
Guess what, `lb.fullServerList` is type `[]*lbpb.Server`.
They are lists of proto.Messages, not proto.Message. (hehehe)